message/validation/genesis/consensus_validation.go

```diff
@@ -10,32 +10,22 @@ import (
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	genesisspecqbft "github.com/ssvlabs/ssv-spec-pre-cc/qbft"
 	genesisspectypes "github.com/ssvlabs/ssv-spec-pre-cc/types"
-	spectypes "github.com/ssvlabs/ssv-spec-pre-cc/types"
 	alanspecqbft "github.com/ssvlabs/ssv-spec/qbft"
-	alanspectypes "github.com/ssvlabs/ssv-spec/types"
 	"golang.org/x/exp/slices"
 
 	"github.com/ssvlabs/ssv/protocol/v2/qbft/roundtimer"
 	ssvtypes "github.com/ssvlabs/ssv/protocol/v2/types"
 )
 
 func (mv *messageValidator) validateConsensusMessage(
-	share *ssvtypes.GenesisSSVShare,
+	share *ssvtypes.SSVShare,
 	signedMsg *genesisspecqbft.SignedMessage,
 	messageID genesisspectypes.MessageID,
 	receivedAt time.Time,
 	signatureVerifier func() error,
 ) (ConsensusDescriptor, phase0.Slot, error) {
 	var consensusDescriptor ConsensusDescriptor
 
-	if mv.operatorDataStore != nil && mv.operatorDataStore.OperatorIDReady() {
-		if mv.inCommittee(share) {
-			mv.metrics.InCommitteeMessage(alanspectypes.MsgType(spectypes.SSVConsensusMsgType), mv.isDecidedMessage(signedMsg))
-		} else {
-			mv.metrics.NonCommitteeMessage(alanspectypes.MsgType(spectypes.SSVConsensusMsgType), mv.isDecidedMessage(signedMsg))
-		}
-	}
-
```

Comment:
can you example how come its not needed anymore? OperatorIDReady used to wait here until we have our operator ID (from db or sync)

Reply:
It's only used for metric if message belongs to committee which we don't use as far as I know (some time ago I added it temporarily to debug something and didn't delete it). Also, we don't have this metric in the new message validation, so I deleted it here too. If you think we need it, I can add it to both